src/pages/staff/dashboard/LearningPage.jsx

```
@@ -269,6 +249,55 @@ export default function LearningPage() {
       });
     };
   }, [user]);
+
+  // Hydrate saved search (q + tags) on mount (guarded by role)
+  useEffect(() => {
+    if (!user || user.role !== 'staff') return;
+    try {
+      const raw = localStorage.getItem(STORAGE_KEY) || '';
+      if (!raw) return;
+      let parsed = null;
+      try { parsed = JSON.parse(raw); } catch (_) { /* backward compat: was plain string */ }
```

ESLint

[error] '_' is defined but never used.

(no-unused-vars)